boards/DCM/Inc/stm32f3xx_hal_conf.h

```diff
@@ -5,7 +5,7 @@
   ******************************************************************************
   * @attention
   *
-  * <h2><center>&copy; COPYRIGHT(c) 2022 STMicroelectronics</center></h2>
+  * <h2><center>&copy; COPYRIGHT(c) 2023 STMicroelectronics</center></h2>
```

Comment:
Hey @gtaharaedmonds this won't cause any problems will it? I think I saw one of the tests that are automatically run when pushing to a branch flag a difference like this.

Reply:
Confirmed, no problems